x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/http/sender/QueryAndDocsInputs.java

```diff
@@ -21,6 +21,19 @@ public static QueryAndDocsInputs of(InferenceInputs inferenceInputs) {
     }
 
     private final String query;
+    private final List<String> chunks;
+    private final boolean stream;
+
+    public QueryAndDocsInputs(String query, List<String> chunks) {
+        this(query, chunks, false);
+    }
+
+    public QueryAndDocsInputs(String query, List<String> chunks, boolean stream) {
```

Comment:
Why are we calling the values chunks here and inputs in the DocumentsOnlyInputs?